highfidelity/hifi
# Updating highlight textures on Rift Controller

Textures were updated for the Rift controller button highlights from a bright yellow to a bright blue.

Test plan:
1. Using Oculus Rift, enter VR
2. Press ESC to set Away mode
3. Look at controller overlays
4. Ensure all buttons are blue and none are yellow. 
5. Ensure buttons are correct corresponding to their physical counterpart (A, B, X, Y)
